README.md

```diff
@@ -1333,6 +1349,79 @@ created() {
 This will add an additional `SEO` button to your section options (edit, drag, delete), that you can use(by clicking on it) to enable/disable overwriting the metas of your page by the ones of this section, based on the return value from the `seo_management` global-hook described above.
 The sections metas complies to the sections order in the page (following their weights). And so the section with the lowest weight and having a meta (ie. `title`) will be the one overwriting the page title meta even if other sections has SEO enabled and returning a title. 
 
+### Builder Settings:
+
+The library support the ability of having custom css form data that will can be saved into the project metadata. 
```

Comment:
again, why are you referring specifically to custom css ? The form can be for any purpose
And it should also be dynamic for any number of forms not just one